DirectProgramming/DPC++FPGA/include/pipe_utils.hpp

```diff
@@ -0,0 +1,264 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#ifndef __PIPE_ARRAY_HPP__
+#define __PIPE_ARRAY_HPP__
```

Comment:
Should be __PIPE_UTILS_HPP__ (don't forget to also change the comment at the end of the file)